addons/hr_expense/tests/test_expenses.py

```diff
@@ -987,3 +987,34 @@ def test_employee_expense_in_foreign_currency(self):
             expense.account_move_id,
             [{'amount_total': 500.0, 'currency_id': expense.account_move_id.company_currency_id.id}],
         )
+
+    def test_company_expense_sepa_ct_trust_bypass(self):
+        """
+        Ensure company-paid expenses using SEPA CT post without requiring a trusted recipient bank account.
+        This validates the bypass in account.payment.action_post for expense-originated payments.
+        """
+        self.env.ref('base.EUR').active = True
+        bank_journal = self.company_data['default_journal_bank']
+        bank = self.env['res.bank'].create({
+            'name': 'BNP Paribas',
+            'bic': 'GEBABEBB',
+        })
+        bank_journal.write({
+            'bank_id': bank.id,
+            'bank_acc_number': 'BE48363523682327',
+            'currency_id': self.env.ref('base.EUR').id,
+        })
+        sepa_ct_line = bank_journal.outbound_payment_method_line_ids.filtered(lambda l: l.code == 'sepa_ct')
+
```

Comment:
Still need a test to check if the module is installed/the payment method exist
I don't know why you removed that part